s2counter_loader/src/test/scala/org/apache/s2graph/counter/loader/stream/ExactCounterStreamingSpec.scala

```diff
@@ -1,217 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one
- * or more contributor license agreements.  See the NOTICE file
- * distributed with this work for additional information
- * regarding copyright ownership.  The ASF licenses this file
- * to you under the Apache License, Version 2.0 (the
- * "License"); you may not use this file except in compliance
- * with the License.  You may obtain a copy of the License at
- * 
- *   http://www.apache.org/licenses/LICENSE-2.0
- * 
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied.  See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-
-package org.apache.s2graph.counter.loader.stream
-
-import org.apache.s2graph.core.{Management, GraphUtil}
-import org.apache.s2graph.core.mysqls.Label
-import org.apache.s2graph.counter.loader.counter.core.{CounterEtlItem, CounterFunctions}
-import org.apache.s2graph.counter.loader.models.DefaultCounterModel
-import org.apache.s2graph.spark.config.S2ConfigFactory
-import org.apache.s2graph.spark.spark.HashMapParam
-import org.apache.spark.streaming.{Seconds, StreamingContext}
-import org.apache.spark.{SparkConf, SparkContext}
-import org.scalatest.{BeforeAndAfterAll, FlatSpec, Matchers}
-import play.api.libs.json.Json
-import CounterFunctions.HashMapAccumulable
-import s2.counter.core.TimedQualifier.IntervalUnit
-import s2.counter.core._
-import s2.counter.core.v2.{ExactStorageGraph, GraphOperation, RankingStorageGraph}
-import s2.helper.CounterAdmin
-import s2.models.{Counter, DBModel}
-
-import scala.collection.mutable.{HashMap => MutableHashMap}
-import scala.concurrent.ExecutionContext.Implicits.global
-import scala.util.{Failure, Success}
-
-/**
-  * Created by hsleep(honeysleep@gmail.com) on 2015. 11. 19..
-  */
-class ExactCounterStreamingSpec extends FlatSpec with Matchers with BeforeAndAfterAll {
```

Comment:
These test-cases cannot build successfully in current version.
I think that we need to rewrite TC and make process to test the spark related sub-project like loader and s2counter_loader.